pallets/capacity/src/tests/stake_and_deposit_tests.rs

```diff
@@ -255,10 +255,10 @@ fn stake_when_staking_amount_is_greater_than_free_balance_it_stakes_maximum() {
 		assert_ok!(Capacity::stake(RuntimeOrigin::signed(account), target, amount));
 
 		// Check that StakingAccountLedger is updated.
-		assert_eq!(StakingAccountLedger::<Test>::get(account).unwrap().active, 190);
+		assert_eq!(StakingAccountLedger::<Test>::get(account).unwrap().active, 189);
```

Comment:
`reducible_balance` requires +1 for Existential Deposit, so these values are updated in the tests.